pkg/asset/installconfig/ovirt/validaton.go

```diff
@@ -125,3 +127,20 @@ func validateVNICProfile(platform ovirt.Platform, con *ovirtsdk.Connection) erro
 	}
 	return nil
 }
+
+func validURL(val interface{}) error {
+	if urlStr, ok := val.(string); ok {
+		url, err := url.ParseRequestURI(urlStr)
+
+		switch {
```

Comment:
I would recommend use if instead of switch. The switch here is not validate any expression before the cases.

Reply:
I prefer the switch in case we want to add another validation condition. The statement is allowed by the language and hepls to avoid if-else chains.

Reply:
Changed for your way also because we have only one validation condition related to https, so makes sense to change for 'if'.